src/actix/api/cluster_api.rs

```diff
@@ -11,16 +11,40 @@ use storage::dispatcher::Dispatcher;
 use storage::rbac::AccessRequirements;
 use validator::Validate;
 
+use crate::actix::api::cluster_api::request_params::QueryParams;
 use crate::actix::auth::ActixAccess;
 use crate::actix::helpers;
 
-#[derive(Debug, Deserialize, Validate)]
-struct QueryParams {
-    #[serde(default)]
-    force: bool,
-    #[serde(default)]
-    #[validate(range(min = 1))]
-    timeout: Option<u64>,
+mod request_params {
```

Comment:
Moved `QueryParams` to a separate module to prevent direct access to the `timeout` attribute here, and in some other places.
This helps ensuring the timeout limit is properly applied.